src/index.ts

```diff
@@ -82,14 +85,21 @@ export default class OASNormalize {
   }
 
   /**
-   * Bundle up the given OpenAPI or Swagger definition, resolving any external `$ref` pointers in
-   * the process.
+   * Bundle up the given API definition, resolving any external `$ref` pointers in the process.
    *
    */
   async bundle() {
     if (this.cache.bundle) return Promise.resolve(this.cache.bundle);
 
     return this.load()
+      .then(schema => {
+        // Postman collections don't support `$ref` pointers so there's nothing for us to bundle.
+        if (utils.isPostman(schema)) {
+          throw new Error('Postman collections cannot be bundled.');
+        }
+
```

Comment:
So in `rdme`, we bundle every definition that comes our way by default: https://github.com/readmeio/rdme/blob/e264f6881a6026a7b540d5e72dedb36c11964cb9/src/lib/prepareOas.ts#L140-L142

I think at the end of the day we just need a function that will always return a validated OpenAPI spec (that bundles all refs if necessary). A few options here:
1. We have a different function to accomplish this (maybe an option in `validate`?)
2. This function could simply return a converted OpenAPI spec if the input is Postman?

Reply:
Ok I've updated `.bundle()` and `.deref()` to automatically upconvert a Postman collection to OAS and then run that through normal bundling and dereferencing workflows. Because Postman doesn't support `$ref` pointers nothing will get modified.